iopipe/contrib/trace/auto_db.py

```diff
@@ -0,0 +1,78 @@
+import collections
+import uuid
+import wrapt
+
+from .util import ensure_utf8
+
+Request = collections.namedtuple(
+    "Request", ["command", "key", "hostname", "port", "connectionName", "db", "table"]
+)
+
+
+def collect_redis_metrics(context, trace, args, connection):
+    command = key = None
+    if len(args) > 0:
+        command = args[0]
+    if len(args) > 1:
+        key = args[1]
+    request = Request(
+        command=ensure_utf8(command),
+        key=ensure_utf8(key),
+        hostname=ensure_utf8(connection.host),
+        port=ensure_utf8(connection.port),
+        connectionName=None,
```

Comment:
From this, I'd assume that named connections aren't supported by python's redis client.  (No problem if it isn't.)

Reply:
Don't see any support for it in the client. Unless this is referring to whether the connection is a pool or not?

Reply:
I initially wrote it to take advantage of the way ioredis.js allows multiple user-defined connections that users can name with arbitrary string values.  Other modules don't necessarily support that, which isn't a deal-stopper, but I could see users appreciating the convenience of seeing named connections if they've been able to define them.